vignettes/get_started.Rmd

```diff
@@ -1,20 +1,18 @@
 ---
-title: "Using cchsflow"
+title: "Get started"
 output: rmarkdown::html_vignette
+date: "`r Sys.Date()`"
```

Comment:
Should we have a date for this vignette? The other vignettes do not have dates on them.

Reply:
let's leave out for now.